resources/js/components/ui/Modal/Modal.module.scss

```diff
@@ -0,0 +1,20 @@
+.modal {
+  [data-part="backdrop"] {
+    @apply fixed inset-0 z-50 bg-black bg-opacity-50;
```

Comment:
z-indexy to zło. O ile element trafia do Portala, który teleportuje DOM node do wspólnego kontenera, to da się to załatwić dosyć łatwo, aby nie pisać z-indexów, a mieć poprawną kolejność wyświetlania. z-index mógłby być tutaj 9999 a i tak nie robi to różnicy (a wiele stron tak ma :D)

Reply:
Wiem że to zło ale zawsze jakoś w końcu była taka sytuacja że trzeba tego było gdzieś użyć, i jeszcze zrozumieć czemu nie działa chociaż daję miljon. A tutaj to w ogóle wziąłem z tego przykładu na stronie zag'a (konkretnie dają 1400 :) ), nie sprawdziłem że niepotrzebne.